defaults/main.yml

```diff
@@ -275,10 +275,10 @@ docker_ferm_dependent_rules:
       }
 
   - type: 'accept'
-    dport: '{{ [ docker_tcp_port ] + docker_custom_tcp_ports }}'
+    dport: '{{ [ docker_tcp_port ] + docker_custom_ports }}'
+    protocol: [ 'tcp', 'udp' ]
     saddr: '{{ docker_tcp_allow }}'
     accept_any: False
     weight: '50'
     role: 'docker'
-    name: 'rest_api_rules'
-
+    name: 'docker_services_rules'
```

Comment:
Change this to `name: 'service_rules`, it will be combined with the `role` parameter to `50_docker_service_rules`.